pkg/health/health_test.go

```diff
@@ -41,7 +41,7 @@ func TestPickServiceResult(t *testing.T) {
 	testMap := getResult([]*api.ServiceEntry{t1, t2, t3})
 	catalog := []*api.ServiceEntry{t1, t2, t3}
 	res, err := selectResult(catalog, nil)
-	Assert(t).AreEqual(err, nil, "catalog is health, kv not present and selectResult returned err")
+	Assert(t).AreEqual(err, nil, "catalog is healthy, kv not present and selectResult returned err")
```

Comment:
in an ideal world these would be in a separate commit reading "fix typos in health test messages" or something